### ISSUE TITLE
As a content admin, I do not want the button to delete a document-place relationship type to appear inline, as it may appear to indicate only removing one relationship and not the type.

### ISSUE THREAD

**Issue by blms** [Princeton-CDH/geniza#1688]
### testing notes (QA)

On the [QA admin site](https://test-geniza.cdh.princeton.edu/admin):
- [x] Navigate to the various places where you can add relationships between all types of entities (Document/Person/Place/Event). It should no longer be possible to delete an entire relationship type from that page. In other words, the X button marked in red in the below image should no longer be present for any of these relationships.

---

This is probably true for other relationship types too. This delete button should probably only appear on the relationship type's own page, not the related records' inlines.

In other words, the button marked red here should be removed from this page:

![Screenshot 2024-12-03 at 11 13 43 AM](https://github.com/user-attachments/assets/ff5c6f21-0859-435f-9355-be5b337e9ef9)

It's worse on wider screens because the buttons will not wrap to the next line, so it's less clear they're only related to the type.

**Comment by kseniaryzhova** [comment]
@blms works as it should, thank you! Closing!
